src/core/logical_layers/renderers/MCDARenderer/components/PopupMCDA.tsx

```
@@ -3,6 +3,14 @@ import { capitalize } from '~utils/common';
 import s from './PopupMCDA.module.css';
 import type { PopupMCDAProps } from '../types';
 
+function formatValue(value: number): string {
+  if (Math.abs(value) < 0.00001 && Math.abs(value) > 0) {
+    return value.toExponential(3);
+  } else {
+    return parseFloat(value.toFixed(3)).toString();
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)